lib/entry-types/parse.js

```diff
@@ -16,7 +16,7 @@ module.exports = function parseHostFileLine(text) {
 
   if (hasContent) {
     const [address, ...hosts] = content.split(/\s+/).filter(Boolean);
-    if (address && hosts && hosts.length && (ip.isV4Format(address) || ip.isV6Format(address))) {
+    if (address && hosts && hosts.length && (net.isIPv4(address) || net.isIPv6(address))) {
```

Comment:
Can you now remove the `ip` dependency or is it used elsewhere?

Reply:
It's also being used for the `isEqual` and `isLoopback` methods